### DISCUSSION TITLE
Separate preprocessing

### DISCUSSION THREAD

**Discussion by kchardon** [CoBrALab/RABIES#373] [Q&A]
Hello,
We want to preprocess our subjects separately and then make a group ICA analysis.
We tried to put the preprocessing folders of the different subjects in one folder but rabies can't deal with this. We wanted to look at the pickle files to see if we could modify them to use all the subjects but we can't open them all.

Is there a way to do the group ICA analysis with subjects preprocessed separately ?

**Comment by gdevenyi** [comment]
Hi,

RABIES uses cohorts to perform unbiased group alignment during registration. Individual subject processing does not conceptually work in this framework.

**Comment by Gab-D-G** [comment]
If you want to run group ICA on subjects processed separately, you will need to run ICA yourself using FSL's MELODIC outside of RABIES.
